applets/clock/Makefile.am

```diff
@@ -136,7 +136,7 @@ $(applet_in_files): $(applet_in_files).in Makefile
 		$< > $@
 
 $(applet_DATA): $(applet_in_files)
-	$(AM_V_GEN) $(MSGFMT) --desktop --keyword Name --keyword Description --template $< -d $(top_srcdir)/po -o $@
+	$(AM_V_GEN) $(MSGFMT) --desktop --keyword= --keyword=Name --keyword=Description --template $< -d $(top_srcdir)/po -o $@
```

Comment:
`--keyword= --keyword=Name`

typo?

Reply:
`--keyword=` do not to use the default keywords which include `Icon`